.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: test
+
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['3.1', '3.0', '2.7']
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Ruby ${{ matrix.ruby-version }}
+        uses: ruby/setup-ruby@ec02537da5712d66d4d50a0f33b7eb52773b5ed1
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+      - name: Install dependencies
+        run: bundle install
+      - name: Run rspec
+        run: bundle exec rspec
+      - name: run pubsub emulator
+        run: docker run -d -p 8085:8085 -it gcr.io/google.com/cloudsdktool/cloud-sdk:321.0.0 gcloud beta emulators pubsub start --host-port=0.0.0.0:8085
```

Update the image name too

```suggestion
        run: docker run -d -p 8085:8085 -it gcr.io/google.com/cloudsdktool/google-cloud-cli:321.0.0 gcloud beta emulators pubsub start --host-port=0.0.0.0:8085
```

---

This test worked with google-cloud-cli:latest so I updated this version to latest instead of using specific version